service/worker/diagnostics/invariant/retry/retry.go

```diff
@@ -85,9 +109,22 @@ func fetchStartedEvent(events []*types.HistoryEvent) *types.WorkflowExecutionSta
 	return nil
 }
 
+func checkRetryPolicy(policy *types.RetryPolicy) IssueType {
+	if policy == nil {
+		return ""
```

Comment:
Unsure if it is an issue - but missing retry policies for activities could be a good sign of issues - due to Cadence not being able to retry it on schedule to start/panic any timeout.

Reply:
yeah its. a grey area. I added it is an issue when an activity has heartbeating configured and no retry policy but havent raised missing retry policy by itself as an issue since I thought it would show up on a lot of workflows and might get ignored out of annoyance